notes/meetings/2021-12/december-15.md

```diff
@@ -0,0 +1,45 @@
+# December 15, 2021 - Meeting Notes
+
+**Faciliators:**
+
+| Name | Abbreviation |
+| - | - | - |
+| Dominik Wilkowski | DW |
+| Esther Semo | ES |
+| Elise Chant | EC |
+
+## Resources
+
+- [Survey Summary Results](assets/survey-results.pdf)
+- [Survey Full Results](assets/survey-results-full.csv)
+
+## Prior
+
+* Maintainers have read proposed RFCs
+
+## Survey findings
+
+* ES to show results
+
+## Prioritisation of actions related to survey findings
+
+* TODO 
+
+## Creating a skills register
+
+* TODO 
```

Comment:
We should create a couple groups and associate them with specific areas.
- A technical group that has their word for technical implementation details
- A design group that has their word in terms of design and UX
- An a11y group that has their word when it comes to a11y
- Perhaps a content group if we end up having enough